src/System.IO.Pipes.AccessControl/tests/NamedPipeTests/NamedPipeTest.AclExtensions.cs

```diff
@@ -40,11 +40,27 @@ public void GetAccessControl_NamedPipe_BeforeWaitingToConnect()
             Assert.Throws<InvalidOperationException>(() => client.SetAccessControl(new PipeSecurity()));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "On Desktop we throw an exception based on the internal PipeStream state. In .NET Core, we do not have access to that member since Pipes.AccessControl is its own assembly.")]
+        public void SetAccessControl_NamedPipeClientStream_Broken()
+        {
+            NamedPipeClientStream client = GetAccessControl_NamedPipeClientStream_Broken();
+            Assert.Throws<InvalidOperationException>(() => client.SetAccessControl(new PipeSecurity()));
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "On Desktop we throw an exception based on the internal PipeStream state. In .NET Core, we do not have access to that member since Pipes.AccessControl is its own assembly.")]
```

Comment:
> I would prefer we not make this case easier because I want us to try and avoid these cases as much as possible.

I'm not following.  This is about reacting to the fact that there is a difference, and if there is a difference that we're going to live with, why then also force us to have duplicated/gross test code?  I don't think making this is easier is going to make us say "ah, well, it was so easy to write the tests to deal with the difference, let's just not fix the product".  Or if it would make us say that, we have way bigger problems ;)